### ISSUE TITLE
release task

### ISSUE THREAD

**Issue by ntanto** [PokemonGoF/PokemonGo-Bot#5625]
### Expected Behavior
release "any" with condition listed

### Actual Behavior
skip
"release": {
      "any": {"release_below_cp": 1500, "release_below_iv": 0.7, "release_below_ivcp": 0, "logic": "and"}
like it doesn't exist

### Your FULL config.json (remove your username, password, gmapkey and any other private info)
"any": {"release_below_cp": 1500, "release_below_iv": 0.7, "release_below_ivcp": 0, "logic": "and"}

### Output when issue occurred
no output at all
like it doesn't exist 

### Other Information
OS: Windows 10
Branch: master 
Git Commit: fef76945022210f4663c091b55750c57684026ec
Python Version: 2.7.12

**Comment by Anakin5** [comment]
read your release rule in plain English and you will find out what is the problem.
Hint: Do you any Pokemon in your bag with cp < 1500 AND iv < 0.7 AND ivcp < 0 ?

**Comment by ntanto** [comment]
here some of that didn't release
Arcanine CP: 816 IV: 0.64
Lapras CP: 127 IV: 0.64
Vaporeon CP: 765 IV: 0.58
Flareon CP: 835 IV: 0.42
Snorlax CP: 1256 IV: 0.67
Snorlax CP: 676 IV: 0.60
Snorlax CP: 578 IV: 0.53
Snorlax CP: 312 IV: 0.51
